conf/modules.config

```
@@ -46,6 +46,10 @@ process {
         ]
     }
 
+    withName: 'UNTAR' {
```

There is no `UNTAR` process is there? They're all aliased in the workflow aren't they?
```suggestion
    withName: 'UNTAR_.*' {
```

---

I think they should all be aliased.

---

It was added to fix up this issue so should be harmless because it is set by default but something specific to DNAnexus https://github.com/nf-core/rnaseq/issues/725

---

It shouldn't work I believe, but I'm getting some weird recursion error in my toy example:
```groovy
workflow.enable.dsl = 2

include { FOO as BAR } from './foo'

workflow {
    BAR ( 'hello' )
}
```
```groovy
process FOO {

    input:
    val message

    script:
    """
    echo $message
    """
}
```
```groovy
process {
    withName: 'FOO' {
        echo = true
    }
}
```
Maybe you're able to check if you have time.

---

Fixed the toy. 
Worryingly, the above example echos, so if you use the original process name, it applies the settings, even to the aliases. So, no, you don't need to change the process selector above, but I feel like this is a bug personally.

---

OK. Convinced this is a bug. 
If I do
```groovy
include { FOO; FOO as BAR } from './foo'
```
then `withName: FOO` applies to both. This is likely not intended ( and to me should only apply to the first `FOO` ).

---

Made a bug report. https://github.com/nextflow-io/nextflow/issues/2490

---

Cool. Fixed [here](https://github.com/nf-core/rnaseq/pull/732/commits/771ec2651c58024774b906eebb2243f0785c41a5) as you suggested.